Sources/BugsnagPerformance/Private/Tracer.mm

```diff
@@ -87,6 +88,9 @@
         case BugsnagPerformanceViewTypeUIKit:   type = @"UIKit"; break;
         default:                                type = @"?"; break;
     }
+    if (appStartupInstrumentation_) {
+        appStartupInstrumentation_->didStartViewLoadSpan(className);
+    }
```

Comment:
If `appStartupInstrumentation_` is created in the constructor, it won't be null here.